src/EduManagementLab.Api/Controllers/CoursesController.cs

```diff
@@ -231,6 +255,17 @@ public class CourseDto
             [Required]
             public DateTime EndDate { get; set; }
         }
+
+        public class MembershipDto
+        {
+            public Guid Id { get; set; }
```

Comment:
Maybe we dont need to expose the membership Id? We have an Id in the database but the API only expose the composite candidate key CourseId and UserId.

You pass CourseId and UserId to AddCourseMembership, UpdateCourseMembershipEnrolledDate and DeleteCourseMembership so the membership Id is not required to use those methods.

Reply:
Removed Id